_includes/pbc/all/install-features/202304.0/install-warehouse-picking-feature.md

```diff
@@ -0,0 +1,534 @@
+                <event>prepare for picking</event>
+            </transition>
+
+            <transition happy="true" condition="PickingList/isPickingStarted">
+                <source>ready for picking</source>
+                <target>picking started</target>
+                <event>start picking</event>
+            </transition>
+
+            <transition happy="true" condition="PickingList/isPickingFinished">
+                <source>picking started</source>
+                <target>picking finished</target>
+                <event>finish picking</event>
+            </transition>
+        </transitions>
+
+        <events>
+            <event name="picking list generation schedule" timeout="1 second" manual="true"/>
+            <event name="generate picking lists" onEnter="true" command="PickingList/GeneratePickingLists"/>
+            <event name="prepare for picking" timeout="1 second" manual="true"/>
+            <event name="start picking" timeout="1 second" manual="true"/>
+            <event name="finish picking" timeout="1 second" manual="true"/>
+        </events>
+    </process>
+</statemachine>
+
+```
+</details>
+
+3. Then, add the `DummyPicking` subprocess to the `DummyPayment01` process as an example.
+Consider OMS configuration using the `DummyPayment01` process as an example:
+
+**config/Zed/oms/DummyPayment01.xml**
+
+```xml
+<?xml version="1.0"?>
+<statemachine
+        xmlns="spryker:oms-01"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd"
+>
+
+    <process name="DummyPayment01" main="true">
+        <subprocesses>
+            <process>DummyPicking</process> <!-- Registering the subprocess. -->
+        </subprocesses>
+
+        <transitions>
+            <!-- We will find the following transition and substitute it with the provided below. -->
+            <!--            <transition happy="true">-->
+            <!--                <source>waiting</source>-->
+            <!--                <target>exported</target>-->
+            <!--                <event>check giftcard purchase</event>-->
+            <!--            </transition>-->
+            <!-- / -->
+
+            <transition happy="true">
+                <source>waiting</source>
+                <target>picking list generation scheduled</target>
+                <event>picking list generation schedule</event>
+            </transition>
+
+            <transition happy="true">
+                <source>picking finished</source>
+                <target>exported</target>
+                <event>finish picking</event>
+            </transition>
+
+        </transitions>
+
+    </process>
+
+    <process name="DummyPicking"
+             file="DummySubprocess/DummyPicking01.xml"/> <!-- Include the subprocess file to be loaded by the state machine. -->
+
+</statemachine>
+```
+
+
+{% info_block warningBox "Verification" %}
+
```

Comment:
Add an introductory sentence to make clear what the user verifies—for example, "Make sure that the OMS transition diagram shows expected transitions:"